src/Icons/src/DependencyInjection/UXIconsExtension.php

```diff
@@ -64,17 +66,23 @@ public function getConfigTreeBuilder(): TreeBuilder
         return $builder;
     }
 
+    /**
+     * @param array<string, mixed> $config
+     */
```

Comment:
Can't this be inferred from `Extension` ?

Reply:
69     Method Symfony\UX\Icons\DependencyInjection\UXIconsExtension::getConfiguration() has parameter $config with no value type specified in iterable type array.  
         💡 See: https://phpstan.org/blog/solving-phpstan-no-value-type-specified-in-iterable-type

Reply:
I mean, typing it in https://github.com/symfony/symfony/blob/1a16ebc32598faada074e0af12a6a698d2964a5e/src/Symfony/Component/DependencyInjection/Extension/ConfigurationExtensionInterface.php#L29

Reply:
I'm all for it, but that's way outside what we can do here ^^